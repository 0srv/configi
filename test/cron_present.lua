
cron.present"test"{
  minute = "25",
  hour = "09",
  job = "/bin/ls"
}

cron.present"cut"{
  minute = "30",
  hour = "11",
  job = "/bin/ls"
}

cron.present"two"{
  minute = "2",
  hour = "4",
  job = "/bin/ls"
}

cron.absent"whoah"{
  job = "/bin/ls"
}


cp = {
  Moe = { minute = "12", hour = "6", job = "/bin/ls" }
}
each(cp, cron.present)

cp = {
  Larry = { minute = "2", hour = "3", job = "/bin/ls" }
}
each(cp, cron.present)

cp = {
  Curly = { minute = "43", hour = "12", job = "/bin/ls" }
}
each(cp, cron.present)

name = "testing"

cron.present(name) {
  minute = "1",
  hour = "6",
  job = "/bin/ls"
}

gang = {
 name = "south"
 minute = "6",
 hour = "7",
 job = "/bin/ls"
}

cron.present(gang.name){
 minute = gang.minute,
 hour = gang.hour,
 job = gang.job,
}


p = {
  job = "/bin/ls"
}

xp = {
  Rec = { minute = "5", hour = "3", job = p.job }
}

for names, jobs in list(xp) do
  cron.present(names)(jobs)
end
