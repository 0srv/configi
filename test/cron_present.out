#	Configi: test
25 09 * * * /bin/ls
#	Configi: cut
30 11 * * * /bin/ls
#	Configi: two
2 4 * * * /bin/ls
#	Configi: Moe
12 6 * * * /bin/ls
#	Configi: Larry
2 3 * * * /bin/ls
#	Configi: Curly
43 12 * * * /bin/ls
#	Configi: testing
1 6 * * * /bin/ls
#	Configi: south
6 7 * * * /bin/ls
#	Configi: Rec
5 3 * * * /bin/ls
